src/pages/mesh/advanced/caching/index.md

```diff
@@ -75,11 +94,27 @@ When the response includes cache-control values, only the [most restrictive valu
 
 <InlineAlert variant="info" slots="text"/>
 
-POST requests are not supported, and GET requests are limited to 2,048 characters.
+GET requests are limited to 2,048 characters.
+
+###  Response headers
+
+The following response headers are returned when caching is enabled:
+
+- `Age` - On cache `HIT`, cached response age in seconds.
+
+- `Cache-Status` - `HIT` or `MISS`.
+
+- `Date` - Date of the response in UTC.
+
+- `Etag` - Unique identifier for a response.
+
+- `Expires` - UTC date when the cached response expires.
+
+- `Last-Modified` - UTC date when the cached response was stored.
```

Comment:
Can we move this to the end, under a section like `Verifying the cached behavior`?

"You can verify the caching behavior of your GraphQL requests based the values of the response headers returned when caching is enabled."